apps/studio/components/interfaces/DataWarehouse/WarehouseCollectionDetail.tsx

```diff
@@ -128,10 +125,16 @@ order by timestamp desc limit ${filters.limit} offset ${filters.offset}
 
   const isLoading = queryLoading || collectionsLoading || isRefetching
 
-  function handleSubmit(e: React.FormEvent<HTMLFormElement>) {
-    e.preventDefault()
-    setFilters({ ...filters, search })
-    refetch()
+  function refreshResults(e?: React.FormEvent<HTMLFormElement>) {
+    e?.preventDefault()
+    setFilters({
+      ...filters,
+      search,
+      interval: {
+        to: () => dayjs().toISOString(),
```

Comment:
Looking at this, looks like the intended effect and maybe the bug you're fixing in the first place 😄 but I found myself a little confused when I had selected a concrete date in the past, and refreshing changed the selected time period.

I realize that's confusing so here's what I mean.

What I expect to happen:

(1) If I have an open-ended interval to current time (e.g., last hour, last week, date range selected that includes the current date), I expect refreshing to update the end time for that range to the current time, which it does 👍🏼 

(2) But if I have a close-ended interval ending at some time in the past (e.g., September 3 - September 4, today being September 11), I don't expect refreshing to change my interval to September 3 - September 11

Not a blocker and happy with it if this is the intended UX 😄